### DISCUSSION TITLE
"= new();"

### DISCUSSION THREAD

**Discussion by Urmel** [nemethviktor/GeoTagNinja#50] [General]
Hi @nemethviktor,
while rebasing, I have to deal with a concurrent modification:
In the first manage favorites PR, you changed the DirectoryElements Property:
![grafik](https://user-images.githubusercontent.com/1786902/211195983-b30854ac-24de-4556-ae23-cdded8e751cc.png)

Final line thus is:
> public DirectoryElementCollection DirectoryElements { get; } = new();

I have a couple of questions regarding this:
Was there trouble with the previous one, where the private variable was visible to the private code (set-ability?)?
And: I don't understand what the "= new();" does, and actually have trouble googleing that. Could you give me a hint, please?
Thanks!!

**Comment by nemethviktor** [comment]
I think that's just resharper applying auto code logic. Feel free to undo
if preferred.

This message has been brought to you by my Huawei P30 Pro and myself. Any
typos blame the phone.

**Reply by Urmel** [reply]
Thanks for the quick reply!
Na - will keep it. Just wanted to learn something ;)
